src/App.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import { ThemeContext } from "./context/providers/ThemeContext";
+import { Home } from "./Pages/Home";
+import { Login } from "./Pages/Login";
+import { SignUp } from "./Pages/SignUp";
+
+function App() {
+
+  let [theme, setTheme] = useState("light");
+  return (
+    <BrowserRouter>
+      <ThemeContext.Provider value={{ theme, setTheme }}>
```

Comment:
put this provider back in src/context/provider/ThemeContextProvider.js

So we just need to write only
<ThemeContextProvider>
...
</ThemeContextProvider>

 and make this file little cleaner